src/main/java/org/sagebionetworks/bridge/exporter3/results/AssessmentResultSummarizer.kt

```diff
@@ -0,0 +1,69 @@
+package org.sagebionetworks.bridge.exporter3.results
+
+import org.sagebionetworks.assessmentmodel.AnswerColumn
+import org.sagebionetworks.assessmentmodel.AssessmentResult
+import org.sagebionetworks.assessmentmodel.serialization.Serialization
+import org.sagebionetworks.assessmentmodel.toFlatAnswers
+import org.sagebionetworks.assessmentmodel.toFlatAnswersDefinition
+import org.sagebionetworks.bridge.rest.model.Assessment
+import org.sagebionetworks.bridge.rest.model.AssessmentConfig
+import org.slf4j.LoggerFactory
+
+class AssessmentResultSummarizer(private val assessment: Assessment, private val assessmentConfig: AssessmentConfig): AssessmentSummarizer {
+
+    init {
+        assert(assessment.frameworkIdentifier == FRAMEWORK_IDENTIFIER)
+    }
+
+    override val resultFilename: String
+        get() = "assessmentResult.json"
+
+    override fun canSummarize(assessment: Assessment): Boolean {
+        return assessment.frameworkIdentifier == "health.bridgedigital.assessment"
```

Comment:
```suggestion
        return assessment.frameworkIdentifier == FRAMEWORK_IDENTIFIER
```
This was probably my fault.